pom.xml

```diff
@@ -50,7 +50,7 @@
         <dep.packaging.version>${dep.airlift.version}</dep.packaging.version>
         <dep.slice.version>0.24</dep.slice.version>
         <dep.aws-sdk.version>1.9.40</dep.aws-sdk.version>
-        <dep.tempto.version>1.12</dep.tempto.version>
+        <dep.tempto.version>1.15-SNAPSHOT</dep.tempto.version>
```

Comment:
yes. this is temporary. Once we do a tempto release with the tempto changes, I will update it.